src/net.cpp

```diff
@@ -290,9 +291,9 @@ bool SeenLocal(const CService& addr)
 {
     {
         LOCK(cs_mapLocalHost);
-        if (mapLocalHost.count(addr) == 0)
-            return false;
-        mapLocalHost[addr].nScore++;
+        const auto it = mapLocalHost.find(addr);
+        if (it == mapLocalHost.end()) return false;
+        it->second.nScore++;
```

Comment:
prefer prefix operator per developer-notes.md
```suggestion
        ++it->second.nScore;
```

Reply:
Thanks, done.